packages/api/src/routes/metaplex-upload.js

```diff
@@ -0,0 +1,158 @@
+import { HTTPError } from '../errors.js'
+import * as cluster from '../cluster.js'
+import { secrets, database } from '../constants.js'
+import { JSONResponse } from '../utils/json-response.js'
+import { toNFTResponse } from '../utils/db-transforms.js'
+import { parseCid } from '../utils/utils.js'
+import {
+  parseJWTHeader,
+  parseJWT,
+  Base64URL,
+  utf8ToUint8Array,
+} from '../utils/jwt.js'
+import { keyFromDID, verifyEd25519Signature } from '../utils/ed25519.js'
+import { DBClient, DBError } from '../utils/db-client.js'
+
+/**
+ * When >2.5MB, use local add, because waiting for blocks to be sent to
+ * other cluster nodes can take a long time. Replication to other nodes
+ * will be done async by bitswap instead.
+ */
+const LOCAL_ADD_THRESHOLD = 1024 * 1024 * 2.5
+
+// TODO: remove when https://github.com/nftstorage/nft.storage/pull/638 merged
+const db = new DBClient(database.url, secrets.database)
```

Comment:
https://github.com/nftstorage/nft.storage/pull/638 was merged, you can obtain this instance from the `ctx` object.